src/main/java/racingcar/RacingCarGameLogic.java

```diff
@@ -17,15 +21,32 @@ public static RacingCarGameLogic getInstance() {
         return instance;
     }
 
-    public void gameLogic(int numberOfCar, int numberOfAttempt) {
-        List<Car> cars = initCars(numberOfCar);
-        IntStream.range(0, numberOfAttempt).forEach(i -> RacingCarGameRule.getInstance().moveCars(cars));
+    public void gameLogic(String nameOfCars, int numberOfAttempt) {
+        List<Car> cars = initCars(nameOfCars);
+
+        for (int i = 0; i < numberOfAttempt; i++) {
+            RacingCarGameRule.getInstance().moveCars(cars);
+            ResultView.getInstance().printMove(cars);
+            ResultView.getInstance().printLineBreak();
```

Comment:
printMove 에서 개행을 처리하면 될 것 같아요
```suggestion
            ResultView.getInstance().printMove(cars);
```